src/english.yml

```diff
@@ -71,6 +71,7 @@ nation_help_6: 'Create a new nation'
 nation_help_7: 'List the king commands'
 nation_help_8: 'Create a new nation'
 nation_help_9: 'List all online residents in nation'
+nation_help_10: 'Teleport to the nations spawn.'
```

Comment:
This will have to go to the end of the language file so that translators will know to translate it.